zh/backup-to-aws-s3-using-br.md

```diff
@@ -198,25 +213,230 @@ Ad-hoc 备份支持快照备份与增量备份。
         provider: aws
         region: us-west-1
         bucket: my-bucket
-        prefix: my-folder
+        prefix: my-full-backup-folder
     ```
 
-在配置 `backup-aws-s3.yaml` 文件时，请参考以下信息：
+在配置 `full-backup-s3.yaml` 文件时，请参考以下信息：
 
 - 自 TiDB Operator v1.1.6 版本起，如果需要增量备份，只需要在 `spec.br.options` 中指定上一次的备份时间戳 `--lastbackupts` 即可。有关增量备份的限制，可参考[使用 BR 进行备份与恢复](https://docs.pingcap.com/zh/tidb/stable/backup-and-restore-tool#增量备份)。
 - Amazon S3 的 `acl`、`endpoint`、`storageClass` 配置项均可以省略。兼容 S3 的存储相关配置，请参考 [S3 存储字段介绍](backup-restore-cr.md#s3-存储字段介绍)。
 - `.spec.br` 中的一些参数是可选的，例如 `logLevel`、`statusAddr` 等。完整的 `.spec.br` 字段的详细解释，请参考 [BR 字段介绍](backup-restore-cr.md#br-字段介绍)。
 - 如果你使用的 TiDB 为 v4.0.8 及以上版本, BR 会自动调整 `tikv_gc_life_time` 参数，不需要配置 `spec.tikvGCLifeTime` 和 `spec.from` 字段。
 - 更多 `Backup` CR 字段的详细解释参考 [Backup CR 字段介绍](backup-restore-cr.md#backup-cr-字段介绍)。
 
+#### 查看快照备份的状态
+
 创建好 `Backup` CR 后，TiDB Operator 会根据 `Backup` CR 自动开始备份。你可以通过如下命令查看备份状态：
 
-{{< copyable "shell-regular" >}}
+```shell
+kubectl get backup -n backup-test -o wide
+```
+
+从上述命令的输出中，你可以找到描述名为 `demo1-full-backup-s3` 的 `Backup` CR 的如下信息，其中 `COMMITTS` 表示快照备份的时刻点：
+
+```
+NAME                   TYPE   MODE       STATUS     BACKUPPATH                              COMMITTS             ...
+demo1-full-backup-s3   full   snapshot   Complete   s3://my-bucket/my-full-backup-folder/   436979621972148225   ...
+```
+
+### 日志备份
+
+你可以使用一个 `Backup` CR 来描述日志备份任务的启动、停止以及清理日志备份数据等操作。日志备份对远程存储访问授权方式与快照备份一致。本节示例创建了名为 `demo1-log-backup-s3` 的 `Backup` CR，对远程存储访问授权方式仅以通过 accessKey 和 secretKey 的方式为例，具体操作如下所示。
+
+#### 启动日志备份
+
+1. 在 `backup-test` 这个 namespace 中创建一个名为 `demo1-log-backup-s3` 的 `Backup` CR。
+
+    ```shell
+    kubectl apply -f log-backup-s3.yaml
+    ```
+
+    `log-backup-s3.yaml` 文件内容如下：
+
+    ```yaml
+    ---
+    apiVersion: pingcap.com/v1alpha1
+    kind: Backup
+    metadata:
+      name: demo1-log-backup-s3
+      namespace: backup-test
+    spec:
+      backupMode: log
+      br:
+        cluster: demo1
+        clusterNamespace: test1
```

Comment:
yes, they can be different, The purpose here is to make backup and tidb in different namespace. I remember it may be blocked when delete namespace if backup and tidb is in the same namespace. This error was encountered when writing azure documents, so i followed this style.

Reply:
OK